src/loaders/zarr_utils/WrappedStore.ts

```diff
@@ -17,8 +17,14 @@ type WrappedStoreOpts<Opts> = {
  * `Readable` is zarrita's minimal abstraction for any source of data.
  * `WrappedStore` wraps another `Readable` and adds (optional) connections to `VolumeCache` and `RequestQueue`.
  */
-class WrappedStore<Opts, S extends Readable<Opts> = Readable<Opts>> implements AsyncReadable<WrappedStoreOpts<Opts>> {
+class WrappedStore<Opts, S extends Readable<Opts> = Readable<Opts>>
```

Comment:
Nit: Does it make sense to change this to WrappedReadOnlyStore now that you have a explicit set function that does nothing?  Or was it the same before this change?

Reply:
I believe calling `set` before would have resulted in an error, so this store was read-only to begin with